README.md

```diff
@@ -1 +1,159 @@
-# Solver
\ No newline at end of file
+<a name="readme-top"></a>
+
```

Comment:
- [ ] Please kindly fix this broken link, I got 404 error; implementing this will make your README file to be more engaging. ℹ️